google-datacatalog-saphana-connector/setup.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import setuptools
+
+release_status='Development Status :: 4 - Beta'
+
+with open('README.md') as readme_file:
+    readme = readme_file.read()
+
+setuptools.setup(
+    name='google-datacatalog-saphana-connector',
+    version='0.1.0',
+    author='Google LLC',
+    description=
+    'Library for ingesting Sap Hana Database metadata into Google Cloud Data Catalog',
+    platforms='Posix; MacOS X; Windows',
+    packages=setuptools.find_packages(where='./src'),
+    namespace_packages=['google', 'google.datacatalog_connectors'],
+    package_dir={'': 'src'},
+    entry_points={
+        'console_scripts': [
+            'google-datacatalog-saphana-connector = google.datacatalog_connectors.saphana:main',
+        ],
+    },
+    include_package_data=True,
+    install_requires=('hdbcli', 'google-datacatalog-rdbms-connector>=0.9.0'),
+    setup_requires=('pytest-runner',),
+    tests_require=('pytest-cov', 'google-datacatalog-connectors-commons-test>=0.6.0'),
+    classifiers=(
+        release_status,
+        'Programming Language :: Python :: 3.7',
```

Comment:
```suggestion
        'Programming Language :: Python :: 3.6',
```
Is it possible to use 3.6 to keep it consistent with the README file?

Reply:
Cloud shell uses 3.7 by default, so I'd suggest changing the README file instead.